### PR TITLE
update quickstart to use visual studio code article

### PR DESCRIPTION
**Pull request by craigshoemaker**

### CODE REVIEW

File: docs-ref-conceptual/index.yml [hunk 1]
```diff
@@ -57,10 +57,10 @@ sections:
     className: cardsM
     columns: 3
     items: 
-    - href: https://docs.microsoft.com/azure/app-service/containers/quickstart-nodejs
-      title: Create a Node.js app
+    - href: https://code.visualstudio.com/docs/nodejs/nodejs-deployment
```
Comment (fiveisprime) [inline docs-ref-conceptual/index.yml]: Can you include the GA tracking parameters in this? `?utm_source=msdocs&utm_medium=node_hub`

**Comment by mikekinsman** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/index.yml](https://github.com/craigshoemaker/azure-docs-sdk-node/blob/vscode-tools-update/docs-ref-conceptual/index.yml) | :white_check_mark:Succeeded | |
[docs-ref-conceptual/media/vs-code-logo.svg](https://github.com/craigshoemaker/azure-docs-sdk-node/blob/vscode-tools-update/docs-ref-conceptual/media/vs-code-logo.svg) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201801171640444268-251%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EFsK508GEh%2FZXxVH84qgH3lP5ihP%2FSKczY%2BxFHOlqrI%3D&st=2018-01-17T16%3A47%3A27Z&se=2018-02-17T16%3A52%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/index.yml (hunk 1 above)
Comment (craigshoemaker) [reply docs-ref-conceptual/index.yml]: sure!

**Comment by mikekinsman** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/index.yml](https://github.com/craigshoemaker/azure-docs-sdk-node/blob/vscode-tools-update/docs-ref-conceptual/index.yml) | :white_check_mark:Succeeded | |
[docs-ref-conceptual/media/vs-code-logo.svg](https://github.com/craigshoemaker/azure-docs-sdk-node/blob/vscode-tools-update/docs-ref-conceptual/media/vs-code-logo.svg) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201801181519418382-251%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=HH99Q1jnpeHci7MB8ym784any8LVxVzWsy2RhNEXHGU%3D&st=2018-01-18T15%3A25%3A07Z&se=2018-02-18T15%3A30%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by craigshoemaker** [comment]
#sign-off

**Pull request merged**